scripts/backport

```diff
@@ -20,6 +22,9 @@ PR_NUMBER="${1}"
 shift 1
 TO_BRANCHES="$@"
 
+# Determine the remote to use
+git remote -v show | grep "https://github.com/[Dd]ata[Dd]og/dd-trace-py (push)" | read REMOTE _
```

Comment:
Should we be instead doing all these steps from a temporary checkout?

Reply:
🤔 I think the `git remote` command depends on the repository configuration, so shouldn't depend on which branch we currently are. Do you have a specific concern in mind?